wifi.go

```diff
@@ -188,6 +221,9 @@ type BSS struct {
 
 	// The status of the client within the BSS.
 	Status BSSStatus
+
+	// BSSLoad elements
```

Comment:
Can you expand on this struct documentation a little? I'm not sure how "elements" applies as it's not a list of things.